clerk/organizations.go

```diff
@@ -161,3 +161,60 @@ func (s *OrganizationsService) CreateInvitation(params CreateOrganizationInvitat
 	_, err := s.client.Do(req, &organizationInvitation)
 	return &organizationInvitation, err
 }
+
+type ListMembershipsParams struct {
+	OrganizationID string
+	Limit          *int
+	Offset         *int
+}
+
+type publicUserData struct {
+	FirstName       *string `json:"first_name"`
+	LastName        *string `json:"last_name"`
+	ProfileImageURL string  `json:"profile_image_url"`
+	Identifier      string  `json:"identifier"`
+}
+
+type organizationPublicUserData struct {
+	publicUserData
+	UserID string `json:"user_id"`
+}
+
+type OrganizationMembership struct {
+	Object          string          `json:"object"`
+	ID              string          `json:"id"`
+	PublicMetadata  json.RawMessage `json:"public_metadata"`
+	PrivateMetadata json.RawMessage `json:"private_metadata,omitempty"`
+	Role            string          `json:"role"`
+	CreatedAt       int64           `json:"created_at"`
+	UpdatedAt       int64           `json:"updated_at"`
+
+	Organization   *Organization               `json:"organization"`
+	PublicUserData *organizationPublicUserData `json:"public_user_data,omitempty"`
```

Comment:
:wrench: If we're returning the `PublicUserData` then we should also export the type. Something like `clerk.OrganizationPublicUserData` or `clerk.OrganizationMembershipPublicUserData`. 

These will never be `null`, so the field should not be a pointer and there's no need to `omitempty` here.